_config.yml

```diff
@@ -183,6 +183,15 @@ authors:
     twitter: Xtina_Starr
     github: xtina-starr
 
+  chungyi:
+    name: Chung-yi Chi
```

Comment:
Can I have a capital "Y", like "Chung-Yi Chi"? In case it has to match with my passport. 😝

Reply:
Yes!